fastchat/model/model_adapter.py

```diff
@@ -250,6 +260,7 @@ def remove_parent_directory_name(model_path):
 
 class VicunaAdapter(BaseAdapter):
     "Model adapater for vicuna-v1.1"
+    use_fast_flag = False
```

Comment:
```suggestion
    use_fast_tokenizer = False
```

1. Set it to false in the base class
2. For child classes, if it is the same as base class, do not need to declare it